src/field_info/types.rs

```diff
@@ -10,7 +10,7 @@ pub struct FieldInfo {
 }
 
 bitflags! {
-    #[derive(Clone, Debug)]
+    #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
```

Comment:
I elected to put the `derive`s list in the order that it appears in the `bitflags`@1.3.2 documentation, and then append `Debug`. If you prefer alphabetical order or some other order, feel free to change it or let me know for next time.